include/ringmesh/mesh/mesh.h

```diff
@@ -827,6 +830,7 @@ namespace RINGMesh
             cell_facet_nn_search_{};
         mutable std::unique_ptr< NNSearch< DIMENSION > > cell_nn_search_{};
         mutable std::unique_ptr< VolumeAABBTree< DIMENSION > > cell_aabb_{};
+        mutable AttributesManager cell_attributes_manager_;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make this member variable "private". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3656)